composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php" : "^7.1",
+        "php" : "^7.1|^8.0",
```

Comment:
shouldn't it be "^7.1 || ^8.0"

Reply:
according to composer docs it should indeed be the double ||
https://getcomposer.org/doc/articles/versions.md#version-range